packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1687,17 +1687,17 @@ export class AccountHandler {
         } = request.app.ua;
 
         // Since the only way to reach this point is clicking a
-        // link from the user's email, we create a verified sessionToken
-        // **unless** the user has a TOTP token.
-        tokenVerificationId = hasTotpToken ? await random.hex(16) : null;
+        // link from the user's email and verifying TOTP if they have it,
```

Comment:
This comment is outdated, users no longer click a link to start the password reset 🙃